docs/code-search/code-navigation/how-to/index_a_typescript_and_javascript_repository.mdx

```diff
@@ -93,28 +98,38 @@ Creating one-off indexes and uploading them is valuable as a proof of concept, b
 The steps here are similar to those in the previous GitHub Actions example.
 
 1. Install `scip-typescript`.
-   ```sh
+
+```sh
    npm install -g @sourcegraph/scip-typescript
-   ```
+```
+
 2. Install the Sourcegraph CLI.
-   ```
+
+```sh
    curl -L https://sourcegraph.com/.api/src-cli/src_linux_amd64 -o /usr/local/bin/src
    chmod +x /usr/local/bin/src
-   ```
-   The exact invocation may change depending on the OS and architecture. See the [`src-cli` README](https://github.com/sourcegraph/src-cli#installation) for details.
+```
+
+The exact invocation may change depending on the OS and architecture. See the [`src-cli` README](https://github.com/sourcegraph/src-cli#installation) for details.
+
 3. `cd` into your project's root (which contains `package.json`/`tsconfig.json`) and run the following:
-   ```sh
+
+```sh
    # Enable (1) type-checking code used from external packages and (2) cross-repo navigation
    # by installing dependencies first with npm or yarn
    npm install
    scip-typescript index # for TypeScript projects
-   ```
-   If you are indexing a JavaScript codebase or a project using Yarn workspaces, tweak the `scip-typescript` invocation as documented in the [Optional scip-typescript flags](#optional-scip-typescript-flags) section.
+```
+
+If you are indexing a JavaScript codebase or a project using Yarn workspaces, tweak the `scip-typescript` invocation as documented in the [Optional scip-typescript flags](#optional-scip-typescript-flags) section.
```

Comment:
This sentence is part of the previous list item/bullet, please do not remove the indentation indicating that.